var/spack/repos/builtin/packages/py-pygtk/package.py

```diff
@@ -36,8 +36,8 @@ class PyPygtk(AutotoolsPackage):
     depends_on("libffi")
     depends_on('cairo')
     depends_on('glib')
-    depends_on('gtkplus')
-    depends_on('py-pygobject')
+    depends_on('gtkplus@2.24:2.99')
+    depends_on('py-pygobject@2.28:2.99')
```

Comment:
This looks correct... that py-gtk is GTK2-only.  Can you add a comment (here and in the docstring at the top) pointing users to `py-pygobject` if they want to do Python-based GTK3?